docs/api-testing/composer.md

```diff
@@ -892,11 +893,19 @@ There are two types of input data sets you can use:
 </table>
 
 ### Unit View
-These buttons switch between the Input Set and Unit views.
-
-<br/>
+These buttons switch between the Input Set and Unit views.<br/>
 <img src={useBaseUrl('img/api-testing/unitView.png')} alt="Unit View"/>
 
+## Test Options
+
+Once you've generated your tests in the Composer, you can manage them from the **Tests** tab. In your project, on the **Tests** tab, hover your mouse over the test line item. You'll see icons that allow you to edit, run, schedule, or delete a test.<br/>
+<img src={useBaseUrl('img/api-fortress/2021/02/scheduleIcons.png')} alt="Schedules Icons"/>
+
+* Pencil icon: Edit the test (opens the **Compose** tab)
+* Play icon: Run the test manually
+* Calendar icon: Open the [test scheduler](/api-testing/schedule-test)
+* Trash icon: Delete the test
+
 ## More Information
 * [Sauce School | API Testing Course and Best Practices](https://training.saucelabs.com/apiTesting/index.html)
```

Comment:
@aliciakrebs can you please remove this link? As per conversation here https://saucelabs.slack.com/archives/C029E067K41/p1659530739437299 the training pages are not updated and there are some issues here and there so I don't think it's good to link those pages